src/seedu/addressbook/data/person/Name.java

```diff
@@ -22,11 +22,29 @@
      * @throws IllegalValueException if given name string is invalid.
      */
     public Name(String name) throws IllegalValueException {
-        String trimmedName = name.trim();
-        if (!isValidName(trimmedName)) {
+        String capitalizedName = capitalizeFirstCharForEveryWord(name.trim());
+        if (!isValidName(capitalizedName)) {
             throw new IllegalValueException(MESSAGE_NAME_CONSTRAINTS);
         }
-        this.fullName = trimmedName;
+        this.fullName = capitalizedName;
+    }
+
+    /**
+     * Format name before adding into address book
```

Comment:
Format -> Formats
Also, javadoc comments, including the tags, should end with a full-stop.